apis/projectcontour/v1/httpproxy.go

```diff
@@ -61,12 +61,16 @@ type Include struct {
 }
 
 // MatchCondition are a general holder for matching rules for HTTPProxies.
-// One of Prefix, Header or QueryParameter must be provided.
+// One of Prefix, Exact, Header or QueryParameter must be provided.
 type MatchCondition struct {
 	// Prefix defines a prefix match for a request.
 	// +optional
 	Prefix string `json:"prefix,omitempty"`
 
+	// Exact defines a exact match for a request.
```

Comment:
would be good to note here that this field cannot be used in an include match condition